Src/Common/Presentation/Model/RecommendationModel.cs

```diff
@@ -39,15 +39,15 @@ public RecommendationModel()
         public void ApplyTo(IRecommendation recommendation)
         {
             recommendation.Type = Type;
-            recommendation.Rating = Rating;
+            recommendation.Rank = Rank;
             //recommendation.User = Name;
-            recommendation.Value = Value;
+            //recommendation.Value = Value;
         }

```

Comment:
Do you mean to delete these lines or uncomment them later? They might be forgotten if they're not called out more conspicuously through an issue or a TODO comment.